licenses/LICENSE_hdrhistogram_bsd-2-clause_cc0

```diff
@@ -1,41 +0,0 @@
-The code in this repository code was Written by Gil Tene, Michael Barker,
```

Comment:
this file had to be added explicitly as a copy of the licence had to be provided, and it's not added in the build-time version.

Reply:
The license is there:

> **9** **Group:** `org.hdrhistogram` **Name:** `HdrHistogram` **Version:** `2.2.2` 
> - **Manifest License**: The 2-Clause BSD License (Not Packaged)
> - **POM Project URL**: [http://hdrhistogram.github.io/HdrHistogram/](http://hdrhistogram.github.io/HdrHistogram/)
> - **POM License**: Creative Commons Legal Code - [https://creativecommons.org/publicdomain/zero/1.0/legalcode](https://creativecommons.org/publicdomain/zero/1.0/legalcode)
> - **POM License**: PUBLIC DOMAIN - [http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/)
> - **POM License**: The 2-Clause BSD License - [https://opensource.org/licenses/BSD-2-Clause](https://opensource.org/licenses/BSD-2-Clause)
> - **Embedded license files**: [HdrHistogram-2.2.2.jar/META-INF/LICENSE.txt](HdrHistogram-2.2.2.jar/META-INF/LICENSE.txt)